### PR TITLE
Client/api integration

### PR DESCRIPTION
**Pull request by jmensch1**
- Jurisdiction and state data are now coming from the dev api
- Map route url contains jurisdictionId, lon, and lat for bookmarking/sharing 
- Updated UI to account for slight differences between live data and the dummy data we were using before
- Fixed console errors, which were preventing the CI from working

Note that we can't put location-markers on the map yet because the locations aren't geocoded in the DB.

Also please add a new var in your `.env` file:
```
REACT_APP_API_URL=https://api.dev.ballotnav.org/public
```

### CODE REVIEW

File: client/web/.example.env [hunk 1]
```diff
@@ -1 +1,2 @@
-REACT_APP_MAPBOX_TOKEN=REDACTED
\ No newline at end of file
+REACT_APP_MAPBOX_TOKEN=REDACTED
+REACT_APP_API_URL=https://api.dev.ballotnav.org/public
```
Comment (jafow) [inline client/web/.example.env:2]: Please update this variable if the react api url is changing
https://github.com/hackforla/ballotnav/blob/client/api-integration/.github/workflows/web-admin.yaml#L35

Comment (jafow) [reply client/web/.example.env:2]: Ditto for the else block

**Review by jafow — APPROVED** [review]
Lgtm 👍

File: client/web/.example.env (hunk 1 above)
Comment (jmensch1) [reply client/web/.example.env:2]: Ok so this stuff is for the public client, not the admin client. The api url for the admin client is staying the same. But yeah we'll need to update the `client.yaml` file at some point so we can get the env vars into the bundle. I think for now I'll just put the API_URL in the code as a default:
```
const BASE_URL = process.env.REACT_APP_API_URL || 'https://api.dev.ballotnav.org/public';
```
We can figure out later how to inject the env vars on the CI server. And I guess we'll probably have to put the mapbox token in a github secret.

**Review by aNullValue — APPROVED** [review]

**Review by tan-nate — COMMENTED** [review]
looks great Jake! wow, what would we do without you?

**Review by tan-nate — APPROVED** [review]

**Pull request merged**